### DISCUSSION TITLE
springboot/springtask

### DISCUSSION THREAD

**Discussion by giscus** [itwanger/tobebetterjavaer-giscus#112] [Announcements]
# springboot/springtask

一份通俗易懂、风趣幽默的Java学习指南，内容涵盖Java基础、Java并发编程、Java虚拟机、Java企业级开发、Java面试等核心知识点。学Java，就认准Java程序员进阶之路

https://tobebetterjavaer.com/springboot/springtask.html

**Comment by ustcjsj** [comment]
https://blog.csdn.net/m0_37479246/article/details/78970809
博客上这张图蛮好的

**Reply by ustcjsj** [reply]
关于三种定时类型（cron  fixedRate fixedDelay ）超时的情形

**Reply by ustcjsj** [reply]
关于异步（Async）https://www.cnblogs.com/wlandwl/p/async.html
